jsk_recognition_msgs/msg/Label.msg

```diff
@@ -0,0 +1,3 @@
+int32 index
```

Comment:
Same comments as in https://github.com/jsk-ros-pkg/jsk_recognition/pull/1981/files#r91411074